app/assets/stylesheets/modules/sitter.scss

```diff
@@ -164,45 +204,93 @@
       font-size: $sitter-headline-font;
       font-weight: $sitter-headline-weight
     }
+  }
+  .hr-1 {
+    width: 100%;
+  }
+  .hr {
+    width: 55%;
+    opacity: 0.3;
+  }
 
-    .hr {
-      opacity: 0.3;
-    }
+  .categories {
+    margin-bottom: 2em;
+    padding-bottom: 2em;
   }
 
- .price {
+  .price {
+    padding-top: 3em;
+    .small-3 {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3